Containerfile.bpfman.openshift

```diff
@@ -4,50 +4,60 @@ ARG BUILDPLATFORM
+    && ${DNF_CMD} install -y gcc openssl-devel \
     && ${DNF_CMD} clean all
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
 
 # Compile bpfman cli, bpfman-ns, and bpfman-rpc binaries
 RUN mkdir -p bin/
+RUN cargo build --release
 RUN if [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
-      cargo build --release --target x86_64-unknown-linux-gnu && \
+      # ${DNF_CMD} install -y gcc-x86_64-linux-gnu && \
+      # CARGO_TARGET_AMD64_UNKNOWN_LINUX_GNU_LINKER=x86_64-linux-gnu-gcc cargo build --release --target x86_64-unknown-linux-gnu && \
       cp target/x86_64-unknown-linux-gnu/release/bpfman bin/. && \
       cp target/x86_64-unknown-linux-gnu/release/bpfman-ns bin/. && \
       cp target/x86_64-unknown-linux-gnu/release/bpfman-rpc bin/.; \
     elif [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
-      apt-get install -y gcc-aarch64-linux-gnu && \
-      rustup target add aarch64-unknown-linux-gnu && \
-      CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc cargo build --release --target aarch64-unknown-linux-gnu && \
+      # ${DNF_CMD} install -y gcc-aarch64-linux-gnu && \
+      # rustup target add aarch64-unknown-linux-gnu && \
+      # CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc cargo build --release --target aarch64-unknown-linux-gnu && \
       cp target/aarch64-unknown-linux-gnu/release/bpfman bin/. && \
       cp target/aarch64-unknown-linux-gnu/release/bpfman-ns bin/. && \
       cp target/aarch64-unknown-linux-gnu/release/bpfman-rpc bin/.; \
     elif [ "$TARGETPLATFORM" = "linux/ppc64le" ]; then \
-      rustup target add powerpc64le-unknown-linux-gnu && \
-      cargo build --release --target powerpc64le-unknown-linux-gnu && \
+      # ${DNF_CMD} install -y gcc-powerpc64le-linux-gnu && \
+      # rustup target add powerpc64le-unknown-linux-gnu && \
+      # CARGO_TARGET_POWERPC64LE_UNKNOWN_LINUX_GNU_LINKER=powerpc64le-linux-gnu-gcc cargo build --release --target powerpc64le-unknown-linux-gnu && \
       cp target/powerpc64le-unknown-linux-gnu/release/bpfman bin/. && \
       cp target/powerpc64le-unknown-linux-gnu/release/bpfman-ns bin/. && \
       cp target/powerpc64le-unknown-linux-gnu/release/bpfman-rpc bin/.; \
     elif [ "$TARGETPLATFORM" = "linux/s390x" ]; then \
-      rustup target add s390x-unknown-linux-gnu && \
-      cargo build --release --target s390x-unknown-linux-gnu && \
+      # ${DNF_CMD} install -y gcc-s390x-linux-gnu && \
+      # rustup target add s390x-unknown-linux-gnu && \
+      # CARGO_TARGET_S390X_UNKNOWN_LINUX_GNU_LINKER=s390x-linux-gnu-gcc cargo build --release --target s390x-unknown-linux-gnu && \
       cp target/s390x-unknown-linux-gnu/release/bpfman bin/. && \
       cp target/s390x-unknown-linux-gnu/release/bpfman-ns bin/. && \
       cp target/s390x-unknown-linux-gnu/release/bpfman-rpc bin/.; \
     fi
 
+FROM redhat/ubi9-minimal
```

Comment:
pls use the same exact ubi image used in the 1st stage so konflux build pipeline can pull successfully

Reply:
u also need to change the entrypoint from `ENTRYPOINT ["./bin/bpfman-rpc", "--timeout=0"]` to `ENTRYPOINT ["./bpfman-rpc", "--timeout=0"]`

Reply:
Done